python/common/org/python/types/Bool.java

```diff
@@ -383,7 +383,13 @@ public boolean __setattr_null(java.lang.String name, org.python.Object value) {
         __doc__ = ""
     )
     public org.python.Object __rtruediv__(org.python.Object other) {
-        throw new org.python.exceptions.NotImplementedError("bool.__rtruediv__() has not been implemented.");
+        try {
+            org.python.types.Int other_val = (org.python.types.Int)other;
+            return new org.python.types.Int(other_val.value == 1 ? 1 : 0).__truediv__(this);
+        } catch (org.python.exceptions.TypeError ae) {
+            throw new org.python.exceptions.TypeError("unsupported operand type(s) for //: 'bool' and '" + other.typeName() + "'");
+        }
+        // throw new org.python.exceptions.NotImplementedError("bool.__rtruediv__() has not been implemented.");
```

Comment:
Can you remove this line? We don't want to leave it in if it's no longer necessary.